### PR TITLE
docs - misc fdw- and foreign table-related doc updates

### PR DESCRIPTION
**Pull request by lisakowen**
in this PR:
- vacuum skips foreign tables
- queries on foreign tables fall back to planner
- postgres fdws should work
- misc edits to "accessing foreign data" page

plan to backport to 6X_STABLE.

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/external/g-foreign.xml [hunk 1]
```diff
@@ -24,6 +28,8 @@
       the Greenplum Database server. Whenever a foreign table is accessed,
       Greenplum Database asks the foreign-data wrapper to fetch data from, or
       update data in (if supported by the wrapper), the remote source.</p>
+    <note>The Pivotal Query Optimizer, GPORCA, does not support foreign tables.
+      A query on a foreign table always falls back to the Postgres query optimizer.</note>
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/external/g-foreign.xml]: Postgres query optimizer -> Postgres planner

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_FOREIGN_TABLE.xml [hunk 2]
```diff
@@ -119,6 +119,12 @@
                 </plentry>
             </parml>
         </section>
+        <section id="section5">
+          <title>Notes</title>
+          <p>The Pivotal Query Optimizer, GPORCA, does not support foreign
+           tables. A query on a foreign table always falls back to the Postgres
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_FOREIGN_TABLE.xml]: Postgres planner

File: gpdb-doc/dita/admin_guide/external/g-foreign.xml (hunk 1 above)
Comment (lisakowen) [reply gpdb-doc/dita/admin_guide/external/g-foreign.xml]: i originally used "Postgres planner", but then i did a search on the docs.  there are 4 refs to this term, but 50+ refs to "Postgres query optimizer".  i assumed that they were referencing the same thing (maybe incorrectly?) and went with the one with more refs.

**Comment by dyozie** [comment]
Darn it - I did the same search before adding my comments, but did so on
the 5.x docs which have 0 instances of "Postgres query optimizer".  Let me
see if I can find the initial note where we tried to standardize on this.
Either way, we may have another story to get the terminology in place.

**Comment by dyozie** [comment]
FYI, https://github.com/greenplum-db/gpdb/pull/7152 is where the original change happened in the code and we agreed to do the same in docs.  The problem is, that change uses both "Postgres planner" and "Postgres query optimizer".  So I guess the main focus was just to get rid of "Legacy".

Since we're mostly using "Postgres query optimizer" let's stick with that here.

File: gpdb-doc/dita/admin_guide/external/g-foreign.xml (hunk 1 above)
Comment (adam8157) [reply gpdb-doc/dita/admin_guide/external/g-foreign.xml]: It should be "Postgres planner" I think, falling back from "The Pivotal Query Optimizer" to "the Postgres query optimizer" seems not right...

Comment (lisakowen) [reply gpdb-doc/dita/admin_guide/external/g-foreign.xml]: ok, using "Postgres Planner".  

docs team will review the doc set and make a similar changes in a separate PR.

Comment (lisakowen) [reply gpdb-doc/dita/admin_guide/external/g-foreign.xml]: @chrishajas or @vraghavan78, can you confirm/review the note above for accuracy?  thanks!

**Comment by berlin-ab** [comment]
What's the latest on this?

**Pull request merged**